### PR TITLE
Keeping the EventID of primary event in cosmic resampling

### PR DESCRIPTION
**Pull request by soleti**
This small edit in the cosmic resampling fcl file allows to keep the EventId of the primary event when resampling, so that we can test for over-sampling.

### CODE REVIEW

File: JobConfig/cosmic/S2Resampler.fcl [hunk 1]
```diff
@@ -17,7 +17,13 @@ physics.producers.g4run.inputs.updateEventLevelVolumeInfos.outInstance: "eventle
 physics.producers.g4run.Mu2eG4CommonCut: @local::Cosmic.Mu2eG4CommonCutCosmicS2
 physics.producers.g4run.SDConfig.preSimulatedHits:  ["cosmicResample:CRV"]
 physics.producers.FindMCPrimary.PrimaryGenIds: ["CosmicCRY", "CosmicCORSIKA"]
+
+resampler.mu2e.writeEventIDs : true
```
Comment (brownd1978) [inline JobConfig/cosmic/S2Resampler.fcl:21]: I think this needs physics.producers.cosmicResample in it

**Review by brownd1978 — APPROVED** [review]
looks good

**Comment by brownd1978** [comment]
@FNALbuild run build test

File: JobConfig/cosmic/S2Resampler.fcl [hunk 2]
```diff
@@ -17,7 +17,11 @@ physics.producers.g4run.inputs.updateEventLevelVolumeInfos.outInstance: "eventle
...
+
+physics.filters.cosmicResample.mu2e.writeEventIDs : true
+
 physics.TriggerPath: [cosmicResample, @sequence::Primary.TriggerSequence]
 outputs.Output.fileName        : "dts.owner.Cosmic.version.sequencer.art"
-// Test on the file with events that deposited lower than 14 MeV in CRV
-//physics.filters.cosmicResample.fileNames: ["/mu2e/app/users/srsoleti/sim.owner.CosmicDSStops.version.sequencer.art"]
+outputs.Output.outputCommands : [@sequence::Primary.KeptProducts, "keep art::EventIDs_*_*_*"]
+
+//physics.filters.cosmicResample.fileNames: ["/mu2e/app/users/srsoleti/sim.owner.CosmicDSStopsCORSIKA.version.sequencer.art"]
```
Comment (oksuzian) [inline JobConfig/cosmic/S2Resampler.fcl:27]: Can you please move this data or persistent?

**Review by oksuzian — APPROVED** [review]

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Pull request merged**